src/networks.js

```diff
@@ -127,16 +127,16 @@ var networks = {
   }
 }
 
-function estimateFee(type) {
-  return function(tx) {
+function estimateFee (type) {
+  return function (tx) {
     var network = networks[type]
     var baseFee = network.feePerKb
     var byteSize = tx.toBuffer().length
 
     var fee = baseFee * Math.ceil(byteSize / 1000)
-    if (network.dustSoftThreshold == undefined) return fee
+    if (network.dustSoftThreshold === undefined) return fee
```

Comment:
delete?  I can't imagine what useful information `null` conveys in this
situation?
On 1 Mar 2015 3:25 pm, "Wei Lu" <notifications@github.com> wrote:

> In src/networks.js
> <https://github.com/bitcoinjs/bitcoinjs-lib/pull/364#discussion_r25563952>
> :
>
> >      var network = networks[type]
> >      var baseFee = network.feePerKb
> >      var byteSize = tx.toBuffer().length
> >
> >      var fee = baseFee * Math.ceil(byteSize / 1000)
> > -    if (network.dustSoftThreshold == undefined) return fee
> > +    if (network.dustSoftThreshold === undefined) return fee
>
> When people explicitly modifies the networks object?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoinjs/bitcoinjs-lib/pull/364/files#r25563952>.
>